db/migrate/20241022144437_add_parent_to_charges_from_plan_parent.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class AddParentToChargesFromPlanParent < ActiveRecord::Migration[7.1]
+  def up
+    now = Time.current
+    Plan.where.not(parent_id: nil).includes(:charges, parent: :charges).find_each do |plan|
```

Comment:
To avoid issues with future fresh install if the Plan model definition changes, you should redefine the Plan model and its relation directly in this migration class. It will make sure that the Plan model you are using is loaded with the expected attributes.

You can refer to https://github.com/getlago/lago-api/blob/main/db/migrate/20240603095841_refresh_cached_aggregations.rb if you need an example

Reply:
ah, right
thanks!